src/import/pull-raw.c

```diff
@@ -478,6 +478,9 @@ static void raw_pull_job_on_finished(PullJob *j) {
         } else if (j == i->settings_job) {
                 if (j->error != 0)
                         log_info_errno(j->error, "Settings file could not be retrieved, proceeding without.");
+        } else if (j == i->signature_job && i->checksum_job->style == VERIFICATION_PER_FILE) {
+                j -> state = PULL_JOB_DONE;
```

Comment:
weird whitespaces before and after `->`... we don't put those usually, please remove them here too